database/resources/blogs.json

```diff
@@ -117,5 +117,12 @@
         "url": "https://blog.openreplay.com/",
         "category": "resources",
         "subcategory": "blogs"
+    },
+    {
+        "name": "Girls Who Code",
+        "description": "Girls Who Code is an international nonprofit organization that strives to close the gender gap in technology . It aims to support and increase the participation of women in computer science and technology.",
+        "url": "https://girlswhocode.com/",
```

Comment:
[Proofreading]
```suggestion
        "url": "https://girlswhocode.medium.com/",
```

Reply:
sorry for the error , I won't happen again . I have updated it